src/openforms/contrib/bag/tests/test_client.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
+
+import requests_mock
+from zds_client import ClientError
+
+from ..client import BAGClient
+from .base import BagTestMixin
+
+
+class BAGClientTests(BagTestMixin, TestCase):
+    @requests_mock.Mocker()
+    def test_client_returns_street_name_and_city(self, m):
+        m.get(
+            "https://bag/api/schema/openapi.yaml?v=3",
+            status_code=200,
+            content=self.load_binary_mock("bagapiprofileoas3.yaml"),
+        )
```

Comment:
this should be using `zgw_consumers.test.mock_service_oas_get` instead: https://github.com/maykinmedia/zgw-consumers/blob/master/zgw_consumers/test/__init__.py#L6